<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddClient</class>
 <widget class="QDialog" name="AddClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Agregar cliente/fábrica</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="newButton">
        <property name="text">
         <string>Nuevo</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="existingButton">
        <property name="text">
         <string>Existente</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="clientLayout">
     <item>
      <widget class="QWidget" name="newWidget" native="true">
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Código</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="codeLineEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>CI/RIF</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="cirifLineEdit"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="nameLineEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Nombre</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Tel 1</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QCheckBox" name="saveAsFactoryBox">
          <property name="text">
           <string>Fábrica</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="telLineEdit"/>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="addressLineEdit"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Dirección</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="existingWidget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QRadioButton" name="clientButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Cliente</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="factoryButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Fábrica</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Filtrar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="filterLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="clientsTableView">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>Agregar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>newButton</tabstop>
  <tabstop>existingButton</tabstop>
  <tabstop>codeLineEdit</tabstop>
  <tabstop>cirifLineEdit</tabstop>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>telLineEdit</tabstop>
  <tabstop>addressLineEdit</tabstop>
  <tabstop>saveAsFactoryBox</tabstop>
  <tabstop>clientButton</tabstop>
  <tabstop>factoryButton</tabstop>
  <tabstop>filterLineEdit</tabstop>
  <tabstop>clientsTableView</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
